### PR TITLE
Implement preserve_url_name to allow users to opt in to using url_name instead of url_name_orig in import

### PR DESCRIPTION
**Pull request by noisecapella**
This adds an argument in XBundle which LORE can use to import with the url_name attribute without breaking backwards compatibility.

cc @ichuang

### CODE REVIEW

File: input_testdata/mitx.01/about/overview.html [hunk 1]
```diff
@@ -0,0 +1 @@
+hello overview
```
Comment (ShawnMilo) [inline input_testdata/mitx.01/about/overview.html]: Does this matter? (Error icons due to no trailing newline.)

Comment (noisecapella) [inline input_testdata/mitx.01/about/overview.html]: I just copied the export data from the other test and made minor modifications. I didn't touch this file at all

**Comment by ShawnMilo** [comment]
Looks good. I'm just going to test in Python 2 and Python 3 locally. One day this needs a real test suite, and maybe some Travis attention.

File: input_testdata/mitx.01/about/overview.html (hunk 1 above)
Comment (ShawnMilo) [inline input_testdata/mitx.01/about/overview.html]: Yeah, I copied a file from an xbundle in a LORE PR and Giovanni asked for it to be fixed. I was just wondering why it's flagged if it doesn't affect functionality.

**Pull request merged**